app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,34 @@
+class RecipesController < ApplicationController
+  def index
+    @recipes = current_user.recipes
+  end
+
+  def new
+    @recipe = Recipe.new
+  end
+
+  def create
+    @recipe = current_user.recipes.build(recipe_params)
+    respond_to do |format|
+      format.html do
+        if @recipe.save
+          redirect_to recipes_path, notice: 'Recipe created successfully'
+        else
+          redirect_to new_recipe_path, alert: 'Error, Recipe not created!'
+        end
+      end
+    end
+  end
```

Comment:
- [ ] I can't create a new recipe because of this error 
     =>>      NoMethodError in RecipesController#index
please fix it